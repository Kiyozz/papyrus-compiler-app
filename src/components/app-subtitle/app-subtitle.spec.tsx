import { shallow } from 'enzyme'
import React from 'react'
import AppSubtitle from './app-subtitle'

describe('<AppSubtitle>', () => {
  it('renders correctly', () => {
    const component = (
      <AppSubtitle />
    )

    expect(component).toBeTruthy()
  })

  it('matches snapshot', () => {
    const component = shallow((
      <AppSubtitle>TOTO</AppSubtitle>
    ))

    expect(component).toMatchSnapshot()
    expect(component.find('h2')).toHaveLength(1)
  })
})
