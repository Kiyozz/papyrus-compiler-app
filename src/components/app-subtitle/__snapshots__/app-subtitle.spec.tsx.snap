// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppSubtitle> matches snapshot 1`] = `
<h2
  className="app-subtitle"
>
  TOTO
</h2>
`;
